### PR TITLE
Add login screen

### PR DESCRIPTION
**Pull request by emilkowalski**
This PR adds a new login screen. I didn't stretch it to fit the viewport on mobile to avoid 100vh issues on various devices, so it's aligned to the top. It also adds a reusable button component + tweaks `blue-600` color in tailwind to match designs.

### CODE REVIEW

File: dashboard/15-final/app/layout.tsx [hunk 1]
```diff
@@ -16,7 +17,7 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <body className={inter.className}>{children}</body>
+      <body className={clsx(inter.className, 'antialiased')}>{children}</body>
```
Comment (emilkowalski) [inline dashboard/15-final/app/layout.tsx:20]: Better font rendering

File: dashboard/15-final/package.json [hunk 2]
```diff
@@ -31,7 +31,8 @@
   },
   "devDependencies": {
     "@types/bcrypt": "^5.0.0",
-    "dotenv": "^16.3.1"
+    "dotenv": "^16.3.1",
+    "prettier": "^3.0.3"
```
Comment (emilkowalski) [inline dashboard/15-final/package.json:35]: I had to install prettier in order to format correctly 🤔 I suppose you don't have that problem? Might be my setup somehow

File: dashboard/15-final/app/ui/logo.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
+export function Logo() {
```
Comment (emilkowalski) [inline dashboard/15-final/app/ui/logo.tsx:1]: I know @StephDietz put the logo in the public folder, we can do that or make it a component, no strong opinion.

File: dashboard/15-final/app/layout.tsx (hunk 1 above)
Comment (delbaoliveira) [reply dashboard/15-final/app/layout.tsx:20]: Nice!

File: dashboard/15-final/app/ui/logo.tsx (hunk 3 above)
Comment (delbaoliveira) [reply dashboard/15-final/app/ui/logo.tsx:1]: I think we landed on the same direction, let's make it a component :)

File: dashboard/15-final/package.json (hunk 2 above)
Comment (delbaoliveira) [reply dashboard/15-final/package.json:35]: Prettier is controlled from the root of next-learn, you might have to clone the root and run `pnpm i`.

**Review by delbaoliveira — DISMISSED** [review]

**Review by delbaoliveira — APPROVED** [review]

**Pull request merged**